lib/traject/psulib_config.rb

```diff
@@ -52,14 +51,13 @@
   end
 end
 
-# HATHI_MONO_OVERLAP constant is expected to be a three column csv of  oclc number, htid and access info.
-# HATHI_MULTI_OVERLAP constant is expected to be a three column csv of oclc number, ht_bib_key and access info.
-# These are generated by running `rake hathitrust:process_hathi_etas`.
-Traject::Macros::Custom::HATHI_MONO_OVERLAP = hathi_to_hash('mono')
-Traject::Macros::Custom::HATHI_MULTI_OVERLAP = hathi_to_hash('multi')
-
 ATOZ = ('a'..'z').to_a.join('')
 ATOU = ('a'..'u').to_a.join('')
+HATHI_OVERLAP_REPORT = CSV.read(ConfigSettings.hathi_overlap_path, headers: true, col_sep: "\t")
+                           .reject { |hr| hr['access'].nil? }
+                           .reject { |hr| ['2168941'].include?(hr['local_id']) }
```

Comment:
this line is about the excludes list - I'll store this in a constant for the next iteration